tools/mo/openvino/tools/mo/moc_frontend/pipeline.py

```diff
@@ -213,4 +213,56 @@ def shape_to_array(shape: PartialShape):
             input_model.set_partial_shape(place, new_partial_shape)
 
     ngraph_function = moc_front_end.convert(input_model)
+    if argv.framework == "pytorch":
```

Comment:
please add `TODO: finish integration of PyTorch FE into MO` and list tikcet here. I remember @andrei-kochin  already created it.
As I understand the code above should also work ideally and special `pytorch_process_after_convert` not needed